smarts/sstudio/types.py

```diff
@@ -464,6 +464,14 @@ class TrapEntryTactic(EntryTactic):
     """The speed that the vehicle starts at when the hijack limit expiry emits a new vehicle"""
 
 
+@dataclass(frozen=True)
+class VehicleSpec:
+    """Vehicle specifications"""
+
+    veh_id: str
+    veh_config_type: str
```

Comment:
This is problematic because we have been and are still distinguishing in our (core) code between `veh_type` and `veh_config_type`, where the latter should eventually  be fully encapsulated to just the Sumo types (per Issue #1270).  I do not think we should be exposing the "config" (Sumo) types in Scenario Studio, but to just change the variable name here to `veh_type` would also be a misnomer without further unification in the core.  Thus, I suggest we deal with this as part of the more comprehensive solution that issue #1270 requires.